manifests/c/Cloudflare/cloudflared/2022.6.1/Cloudflare.cloudflared.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Cloudflare.cloudflared
+PackageVersion: 2022.6.1
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+ReleaseDate: 2022-06-15
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/cloudflare/cloudflared/releases/download/2022.6.1/cloudflared-windows-amd64.msi
+  InstallerSha256: F4BF3FF9692E2BE386B9D89509BA85F0C0D9717B341027F795F7806D9FA298DA
```

```suggestion
  InstallerSha256: F4BF3FF9692E2BE386B9D89509BA85F0C0D9717B341027F795F7806D9FA298DA
  ProductCode: "{35E5E858-9372-4449-BF73-1CD6F7267128}"
```
`Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\{35E5E858-9372-4449-BF73-1CD6F7267128}`